Lesson-2

annbayda opened the pull request:
что-то загрузила

annbayda committed:
Lesson-1

VanillaWulf commented on index.html:
```diff
@@ -3,33 +3,33 @@
         <ul class="menu">
-            <li><a href="#">Menu </a></li>
-            <li><a href="#">Man</a></li>
+            <li><a href="#">MENU </a></li>
```
а зачем вы поменяли? у вас было правильно - настройки шрифта, в том числе uppercase делается в css

VanillaWulf replied:
поправьте обратно((

VanillaWulf commented on index.html:
```diff
@@ -39,191 +39,36 @@
-        <main class="content">The brand</main>
-        <p class="content_1">Of luxerious
-        <span class="content_2">Fashion </span>
+        <main class="content">THE BRAND</main>
```
http://htmlbook.ru/html/main

VanillaWulf commented on index.html:
```diff
@@ -39,191 +39,36 @@
-        <p class="content_1">Of luxerious
-        <span class="content_2">Fashion </span>
+        <main class="content">THE BRAND</main>
+        <p class="content_1">OF LUXERIOUS
```
класс не понятно чем отличается 1 и 2  - сделайте понятные классы

VanillaWulf commented on index.html:
```diff
@@ -39,191 +39,36 @@
        </div>
         
 <div class="photo">
+    <img src="./img/Layer%2043.svg" class="photo_1" alt="woman">
```
% замените на -

VanillaWulf commented on index.html:
```diff
@@ -39,191 +39,36 @@
+    <img src="./img/Layer%2043.svg" class="photo_1" alt="woman">
+    <img src="./img/Rectangle%2030.svg" class="photo_2" alt="men">
+    <img src="./img/Rectangle_child.svg" class="photo_3" alt="child">
+    <img src="./img/Rectangle%20_long.svg" class="photo_4" alt="chassures">
```
в названиях обычно вместо пробела используется - 
как правило названия делаются маленькими буквами

VanillaWulf commented on index.html:
```diff
@@ -39,191 +39,36 @@
     </div>
     
       
-     <div class="card-product-first-line">
```
это был список - куда вы все удалили?

annbayda committed:
Add files via upload

annbayda committed:
Add files via upload

annbayda committed:
Lessons-5

annbayda committed:
Add files via upload

annbayda committed:
Merge pull request #5 from annbayda/Lessons-6

Lessons 6

annbayda committed:
Lessons-7

пока успела сделать адаптив на index.html @media (max-width: 768px)

annbayda committed:
Lessons-8

index @media (max-width: 768px) @media (max-width: 480px)
catalog @media (max-width: 768pxpx)

annbayda committed:
Merge pull request #7 from annbayda/lesson-8

Lessons-8
